lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -61,8 +46,15 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
     end
   end
 
-  defp relevant_alerts(alerts) do
-    Enum.filter(alerts, &(&1.effect == :elevator_closure))
+  defp relevant_closure?(closure) do
+    relevant_effect?(closure) and informs_one_facility?(closure)
+  end
+
+  defp relevant_effect?(closure), do: closure.effect == :elevator_closure
```

Comment:
The "closure" renaming might have gone a bit too far here... I think when we're dealing with anything that's literally a `Screens.Alerts.Alert`, we should call that an `alert`. I thought it made sense to specifically call the entity that has an `elevator_name` and `header_text` and etc. a "closure" because the structure is very distinct from an alert and doesn't only include alert data (it also incorporates facility data).